### DISCUSSION TITLE
"You don't have permission to trigger this action" trying to rebuild from web ui

### DISCUSSION THREAD

**Discussion by hugalafutro** [Lissy93/dashy#582] [Help and Support] [answered]
Unsure if bug or some permission problem on my side, but trying to Rebuild the app manually from web ui yields this screen:
![image](https://user-images.githubusercontent.com/30209689/160595907-659c3208-f441-4d67-8664-13327aef39bb.png)

Otherwise the rebuilds happen automatically, I can trigger them from cli or by restarting the container, I just noticed the red text when browsing the options looking for something else.

**Comment by lissy93** [comment, marked as answer]
Do you have authentication configured? If so, you must be logged in, as an admin. 
Also double check that you've not disabled config editing or set `preventWriteToDisk`

**Reply by lissy93** [reply]
https://github.com/Lissy93/dashy/blob/c2559bc7310d42550f49115a8adc7b103af8fd19/src/components/Configuration/RebuildApp.vue#L125-L130

**Reply by hugalafutro** [reply]
Sorry for late reply, these seem to be in order:
![image](https://user-images.githubusercontent.com/30209689/160837012-a62ade12-ba9a-4e62-9329-b2006442b227.png)

Only have 1 user configured and I am logged as it.

**Reply by lissy93** [reply]
That's really weird, I can't recreate this at all. Are you using the latest version (2.0.6)? And if you open in incognito / or clear browser data, does the issue persist?

It's not too major, as rebuilding the app is no longer required for changes to take effect. This feature will probably be removed in a later version. In the meantime, if you really need to rebuild, then you can send the `yarn build` command to your Docker container.

**Reply by hugalafutro** [reply]
I don't actually need to rebuild and the automatic/manual methods work, I just noticed the warning browsing around the options. More importantly though you're right, it does not happen in incognito mode or different browser so it's browser data related.
